cmd/lotus-shed/chain_index.go

```diff
@@ -135,8 +136,10 @@ number of failed RPC calls. Otherwise, it will exit with a zero status.
 			_, _ = fmt.Fprintf(cctx.App.Writer, "%s starting chainindex validation; from epoch: %d; to epoch: %d; backfill: %t; log-good: %t\n", currentTimeString(),
 				fromEpoch, toEpoch, backfill, logGood)
 		}
-
 		totalEpochs := fromEpoch - toEpoch + 1
+		haltNoData := false
+		haltHeight := -1
```

Comment:
we could just use `haltHeight` as the signal without `haltNoData` by checking for `-1`